test/integration_test/migration_dr_actions_test.go

```diff
@@ -0,0 +1,548 @@
+
+	// Create the apps for migration
+	// Start migrationschedule
+	instanceID := "multi-ns-migration-schedule-interval"
+	// appKeys is an array of appKeys that will be used to create the apps
+	mysqlApp := "mysql-1-pvc"
+	mysqlNamespace := fmt.Sprintf("%s-%s", mysqlApp, instanceID)
+	elasticsearchApp := "elasticsearch"
+	elasticsearchNamespace := fmt.Sprintf("%s-%s", elasticsearchApp, instanceID)
+	appKeys := []string{mysqlApp, elasticsearchApp}
+	var err error
+	// Reset config in case of error
+	defer func() {
+		err = setSourceKubeConfig()
+		log.FailOnError(t, err, "Error resetting remote config")
+	}()
+
+	// Schedule multiple apps
+	var preMigrationCtxs []*scheduler.Context
+	for _, appKey := range appKeys {
+		ctxs, err := schedulerDriver.Schedule(instanceID,
+			scheduler.ScheduleOptions{
+				AppKeys: []string{appKey},
+				Labels:  nil,
+			})
+		log.FailOnError(t, err, "Error scheduling task")
+		Dash.VerifyFatal(t, 1, len(ctxs), "Only one task should have started")
+		preMigrationCtxs = append(preMigrationCtxs, ctxs[0].DeepCopy())
+	}
+
+	for _, preMigrationCtx := range preMigrationCtxs {
+		err = schedulerDriver.WaitForRunning(preMigrationCtx, defaultWaitTimeout, defaultWaitInterval)
+		log.FailOnError(t, err, "Error waiting for app to get to running state")
+	}
+
+	sourceDeployments, err := apps.Instance().ListDeployments(mysqlNamespace, metav1.ListOptions{})
+	log.FailOnError(t, err, "error retrieving deployments from %s namespace", mysqlNamespace)
+	Dash.VerifyFatal(t, len(sourceDeployments.Items), 1, fmt.Sprintf("Expected 1 deployment in destination in %s namespace", mysqlNamespace))
+	sourceDeploymentReplicas := *sourceDeployments.Items[0].Spec.Replicas
+	sourceStatefulsets, err := apps.Instance().ListStatefulSets(elasticsearchNamespace, metav1.ListOptions{})
+	log.FailOnError(t, err, "error retrieving statefulsets from %s namespace", elasticsearchNamespace)
+	Dash.VerifyFatal(t, len(sourceStatefulsets.Items), 1, fmt.Sprintf("Expected 1 statefulset in destination in %s namespace", elasticsearchNamespace))
+	sourceStatefulsetReplicas := *sourceStatefulsets.Items[0].Spec.Replicas
+
+	// Create the clusterpair
+	clusterPairNamespace := defaultAdminNamespace
+	if !unidirectionalClusterpair {
+		log.Info("Bidirectional flag is set, will create bidirectional cluster pair:")
+		log.InfoD("Name: %s", remotePairName)
+		log.InfoD("Namespace: %s", clusterPairNamespace)
+		log.InfoD("Backuplocation: %s", defaultBackupLocation)
+		log.InfoD("Secret name: %s", defaultSecretName)
+		err = scheduleBidirectionalClusterPair(remotePairName, clusterPairNamespace, projectIDMappings, defaultBackupLocation, defaultSecretName)
+		log.FailOnError(t, err, "failed to set bidirectional cluster pair: %v", err)
+		err = setSourceKubeConfig()
+		log.FailOnError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+	} else {
+		log.Info("Creating Unidirectional flag is set, will create unidirectional cluster pair:")
```

Comment:
actually for unidirectional case these tests won't work. 
The behaviour is different in failover if cluster pair is absent in destination cluster. We will either have to modify these cases or have different tests for unidirectional altogether.
![FailoverLLD - Page 4](https://github.com/libopenstorage/stork/assets/141731270/f7cf75df-4869-406b-b0c7-70581a3ac7cc)

Reply:
Right, I will make it bidirectional only for these cases.